src/Symfony/Component/Form/FormRendererEngineInterface.php

```diff
@@ -21,11 +21,13 @@
     /**
      * Sets the theme(s) to be used for rendering a view and its children.
      *
-     * @param FormView $view   The view to assign the theme(s) to
-     * @param mixed    $themes The theme(s). The type of these themes
-     *                         is open to the implementation.
+     * @param FormView $view             The view to assign the theme(s) to
+     * @param mixed    $themes           The theme(s). The type of these themes
+     *                                   is open to the implementation.
+     * @param bool     $useDefaultThemes If true, will use default themes specified
+     *                                   in the engine, will be added to the interface in 4.0
      */
-    public function setTheme(FormView $view, $themes);
+    public function setTheme(FormView $view, $themes /*, $useDefaultThemes = true */);
```

Comment:
in all places where this is added, the code should throw a deprecation when the argument is not provided (using func_num_args, see other places in the code base).

Reply:
@nicolas-grekas As per @fabpot, deprecation notice is not needed if the argument is optional?

Reply:
I think the current code is good as is :)